doc/source/release-notes.rst

```diff
@@ -2,6 +2,19 @@
 Release History
 ***************
 
+v1.0.0 (2019-06-03)
+===================
+
+Features
+--------
+* Rewrite approach to logging.
+* Add precision to motor_ph in mini_beamline.
+
+Fixes
+-----
+* Scan field write in server.
```

Comment:
Can you expand on this? I actually can't recall what this was about.

Reply:
@danielballan 
https://github.com/caproto/caproto/commit/314617bd809c1d0ff883814832a39e8709d92e7e
Do you prefer more details in docs?

Reply:
```suggestion
* Fix bug in `scan_rate` that raised errors when it was written to
```